### PR TITLE
Editorial: Close more old non-normative issues

### PR DESCRIPTION
**Pull request by ptomato**
#2471, #2495, #2531

### CODE REVIEW

File: polyfill/lib/intl.mjs [hunk 1]
```diff
@@ -27,37 +31,74 @@ const CAL_ID = Symbol('calendar-id');
...
+      'timeZoneName',
+      'formatMatcher',
+      'dateStyle',
+      'timeStyle'
+    ];
+    options = ES.ToObject(options);
+    const newOptions = ObjectCreate(null);
+    for (const prop of props) {
+      if (ES.HasOwnProperty(options, prop)) {
+        // Avoid triggering setters on Object.prototype
```
Comment (Ms2ger) [inline polyfill/lib/intl.mjs:83]: How would it trigger them with a null prototype?

**Review by Ms2ger — APPROVED** [review]

File: polyfill/lib/intl.mjs (hunk 1 above)
Comment (ptomato) [reply polyfill/lib/intl.mjs:83]: Right. That must have been left over from some intermediate debugging.

File: spec/calendar.html [hunk 2]
```diff
@@ -1216,12 +1216,13 @@ <h1>
...
+        1. Let _ignoredKeys_ be an empty List.
         1. For each element _key_ of _keys_, do
-          1. Add _key_ to _ignoredKeys_.
-          1. If _key_ is *"month"*, add *"monthCode"* to _ignoredKeys_.
-          1. Else if _key_ is *"monthCode"*, add *"month"* to _ignoredKeys_.
-        1. Return the List of _ignoredKeys_' elements.
+          1. Append _key_ to _ignoredKeys_.
+          1. If _key_ is *"month"*, append *"monthCode"* to _ignoredKeys_.
+          1. Else if _key_ is *"monthCode"*, append *"month"* to _ignoredKeys_.
+        1. Remove duplicate elements from _ignoredKeys_.
```
[the remark this answers is not in the document]
Comment (ptomato) [reply spec/calendar.html:1224]: How about this: "NOTE: _ignoredKeys_ may have duplicate elements. Implementations may choose to represent it as a type that stores unique elements."?

[the remark this answers is not in the document]
Comment (ptomato) [reply spec/calendar.html]: We talked offline and settled on "While _ignoredKeys_ can have duplicate elements, this is not intended to be meaningful. This specification only checks whether particular keys are or are not members of the list."

**Pull request merged**